src/Sentry.Log4Net/SentryAppender.cs

```diff
@@ -65,41 +66,48 @@ protected override void Append(LoggingEvent loggingEvent)
                 }
             }
 
-            var exception = loggingEvent.ExceptionObject ?? loggingEvent.MessageObject as Exception;
-            var evt = new SentryEvent(exception)
+            if (loggingEvent.Level > Breadcrumb)
             {
-                Sdk =
+                var exception = loggingEvent.ExceptionObject ?? loggingEvent.MessageObject as Exception;
+                var evt = new SentryEvent(exception)
                 {
-                    Name = Constants.SdkName,
-                    Version = NameAndVersion.Version
-                },
-                Logger = loggingEvent.LoggerName,
-                Level = loggingEvent.ToSentryLevel()
-            };
+                    Sdk =
+                    {
+                        Name = Constants.SdkName,
+                        Version = NameAndVersion.Version
+                    },
+                    Logger = loggingEvent.LoggerName,
+                    Level = loggingEvent.ToSentryLevel()
+                };
 
-            evt.Sdk.AddPackage(ProtocolPackageName, NameAndVersion.Version);
+                evt.Sdk.AddPackage(ProtocolPackageName, NameAndVersion.Version);
```

Comment:
Are these tabs? :P

Reply:
According to [Render Whitespace on GitHub](https://github.com/glebm/render-whitespace-on-github) they're spaces.

![image](https://user-images.githubusercontent.com/831974/61225522-be800900-a6ee-11e9-9f5c-6aa796c3d1c7.png)

Reply:
TIL :)

Reply:
I usually select it and spaces go one at a time (4), while tab would move at once.
I noticed it was 4 spaces but I'm not sure why it's showing as modified. I guess it was a tab before then :)